Exercise-25-JS/index.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+ let weekdayNumber = +prompt("Choose a number 1 to 7", " ");
+
+ if (weekdayNumber == 1) {
+     console.log("Today is Monday");
+    } 
+    else if
+    (weekdayNumber == 2) {
+     console.log("Today is Tuesday");
+    }
+    else if 
+     (weekdayNumber == 3) {
+    console.log("Today is Wednesday");
+    } 
+    else if (weekdayNumber == 4) {
+    console.log("Today is Thursday");
+    } 
+    else if (weekdayNumber == 5) {
+    console.log("Today is Friday");
+    }
+    else if (weekdayNumber == 6) {
+    console.log("Today is Saturday");
+    } 
+    else if (weekdayNumber == 7) {
+    console.log("Today is Sunday");
+    } 
+    else if (weekdayNumber << 8) {
+    console.log("error message: Error: Number not between 1-7")
+    }
+
+    switch (weekdayNumber) {
```

Comment:
What you should do here is declaring the variable on top and setting the value in the case blocks. Let me know if you are still confused

Good job so far in attempting the exercise

Reply:
Fixed!